doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -334,18 +334,25 @@ Binding arguments to scopes
 Effects of :cmd:`Arguments` on unfolding
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+ `simpl never` indicates that a :term:`constant` should never be unfolded by :tacn:`cbn` or
-  :tacn:`simpl`:
++ `simpl never` indicates that a :term:`constant` should not be unfolded by :tacn:`cbn` or
+  :tacn:`simpl` when in head position. Note that in the case of :tacn:`simpl`, the
+  modifier does not apply in position of reduction of the main argument of a `match`, `fix`,
+  primitive projection, or of an unfoldable constant hiding a `match`,
+  `fix` or primitive projection.
```

Comment:
```suggestion
+ `simpl never` indicates that a :term:`constant` should not be unfolded by :tacn:`cbn` or
  :tacn:`simpl` when in head position. Note that in the case of :tacn:`simpl`, the
  modifier does not apply to reduction of the main argument of a `match`, `fix`,
  primitive projection, or of an unfoldable constant hiding a `match`,
  `fix` or primitive projection.
```

"in position of" was confusing; perhaps a literal translation of a French idiom?